### PR TITLE
docs:  focus homepage on discussing what Substrait is

### PR DESCRIPTION
**Pull request by EpsilonPrime**
Moves the design principles behind Substrait into a separate "About Substrait"
page to allow the homepage to focus on the task of explaining what Substrait is.

### CODE REVIEW

File: site/docs/index.md [hunk 1]
```diff
@@ -5,62 +5,33 @@ title: Home
 
 
 
+## What is Substrait?
 
+Substrait enables interoperability between different implementations of the data compute ecosystem.  This allows one to make the best solution for any problem by choosing the best components.  It doesn't matter if they use different languages or run on different operating systems -- they just work together.
```
Comment (ianmcook) [inline site/docs/index.md:10]: @EpsilonPrime do you want to say what Substrait the _project_ is, or what Substrait the _format_ is?

As is, it's more describing the project—which is fine but maybe adds a layer of indirection for people who are trying to grok how Substrait fits into a system architecture.

**Comment by EpsilonPrime** [comment]
To make it easier for others to see what the new homepage will look like, here's a screenshot:

image

File: site/docs/index.md (hunk 1 above)
Comment (ianmcook) [inline site/docs/index.md:10]:
```suggestion
Substrait is a format for describing compute operations on structured data. It is designed for interoperability across different languages and systems.
```

Comment (ianmcook) [reply site/docs/index.md:10]: This answers the question "what is Substrait" with a noun phrase (it's a format) and it principally describes the format, not the project.

Comment (EpsilonPrime) [reply site/docs/index.md]: Thanks for getting that feedback.  I've updated the PR with this in mind.

File: site/docs/about.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+---
+title: About Substrait
+---
+# Substrait: Cross-Language Serialization for Relational Algebra
+
+
+
+## Project Vision
+
+Create a well-defined, cross-language [specification](spec/specification) for data compute operations. This includes a declaration of common operations, custom operations and one or more serialized representations of this specification. The spec focuses on the semantics of each operation and a consistent way to describe.
```
Comment (ianmcook) [inline site/docs/about.md:10]: I think this is clearer:
```suggestion
The Substrait project aims to create a well-defined, cross-language [specification](spec/specification) for data compute operations. The specification declares a set of common operations, defines their semantics, and describes their behavior unambiguously. The project also defines extension points and serialized representations of the specification.
```

**Review by ianmcook — APPROVED** [review]

File: site/docs/index.md [hunk 3]
```diff
@@ -5,62 +5,34 @@ title: Home
...
+Substrait is a format for describing compute operations on structured data. It is designed for interoperability across different languages and systems.
 
-## Project Vision
 
-Create a well-defined, cross-language [specification](spec/specification) for data compute operations. This includes a declaration of common operations, custom operations and one or more serialized representations of this specification. The spec focuses on the semantics of each operation and a consistent way to describe.
 
-In many ways, the goal of this project is similar to that of the Apache Arrow project. Arrow is focused on a standardized memory representation of columnar data. Substrait is focused on what should be done to data.
+## How does it work?
 
+Substrait provides a well-defined, cross-language [specification](spec/specification) for data compute operations.  This includes a consistent declaration of common operations, custom operations and one or more serialized representations of this specification.  The spec focuses on the semantics of each operation.
```
Comment (jacques-n) [inline site/docs/index.md:16]: I'd an additional sentence here along the lines of "In addition the format and specification, the Substrait community also includes a number of libraries and other useful tools [to get started](/do_we_have_a_list_of_tools_like_validator_isthmus_etc)

Comment (EpsilonPrime) [reply site/docs/index.md]: Done.

File: site/docs/index.md [hunk 4]
```diff
@@ -5,62 +5,34 @@ title: Home
...
-
 
+## Example Use Cases
 
-## Why the name Substrait?
+* Communicate a compute plan between a SQL parser and an execution engine (e.g. Calcite SQL parsing to Arrow C++ compute kernel)
+* Serialize a plan that represents a SQL view for consistent use in multiple systems (e.g. Iceberg views in Spark and Trino)
+* Submit a plan to different execution engines (e.g. Datafusion and Postgres) and get a consistent interpretation of the semantics.
+* Create an alternative plan generation implementation that can connect an existing end-user compute expression system to an existing end-user processing engine (e.g. Pandas operations executed inside SingleStore)
+* Build a pluggable plan visualization tool (e.g. D3 based plan visualizer)
```
Comment (westonpace) [inline site/docs/index.md:37]: This is repeated here and in the about.md page. Is that intentional?

File: site/docs/index.md [hunk 5]
```diff
@@ -5,62 +5,34 @@ title: Home
...
 
-## Project Vision
 
-Create a well-defined, cross-language [specification](spec/specification) for data compute operations. This includes a declaration of common operations, custom operations and one or more serialized representations of this specification. The spec focuses on the semantics of each operation and a consistent way to describe.
 
-In many ways, the goal of this project is similar to that of the Apache Arrow project. Arrow is focused on a standardized memory representation of columnar data. Substrait is focused on what should be done to data.
+## How does it work?
 
+Substrait provides a well-defined, cross-language [specification](spec/specification) for data compute operations.  This includes a consistent declaration of common operations, custom operations and one or more serialized representations of this specification.  The spec focuses on the semantics of each operation.  In addition to the format and specification, the Substrait community also includes a number of libraries and other [useful tools to get started](/tools/producer_tools/).
```
Comment (westonpace) [inline site/docs/index.md:16]: > This includes a consistent declaration of common operations, custom operations and one or more serialized representations of this specification.

I find this sentence a little hard to follow.  Maybe shorter sentences?

```
Substrait allows for common operations to be expressed consistently in different systems.  It also contains many extension points for expressing custom operations.  The specification can be serialized into various formats.
```

> In addition to the format and specification

What is "the format"?  I'm not sure that is defined anywhere.

File: site/docs/index.md (hunk 4 above)
Comment (EpsilonPrime) [reply site/docs/index.md:37]: Removed from the about section (the examples being here should help elucidate the project better).

File: site/docs/index.md (hunk 5 above)
Comment (EpsilonPrime) [reply site/docs/index.md]: Updated the text a bit, I hope I got what you were looking at.

File: site/docs/index.md [hunk 6]
```diff
@@ -5,62 +5,34 @@ title: Home
...
-* [Apache Arrow](https://arrow.apache.org/): The Arrow format for data is what the Substrait specification attempts to be for compute expressions. A key goal of Substrait is to enable Substrait producers to execute work within the Arrow Rust and C++ compute kernels.
+## Benefits
 
+* Avoids every system needing to create a communication method between every other system -- each system merely supports ingesting and producing Substrait and it instantly becomes a part of the greater ecosystem.
+* Makes every part of the system upgradable.  There's a new query engine that's ten times faster?  Just plug it in!
+* Enables heterogeneous environments -- run on a cluster of an unknown set of execution engines!
+* The text version of the Substrait plan allows you to quickly see how a plan functions without needing a visualizer (although there are Substrait visualizers as well!).
```
Comment (richtia) [inline site/docs/index.md:27]: Should we hold off on mentioning a visualizer until we can provide a link/more information on it?

**Review by richtia — APPROVED** [review]

File: site/docs/index.md (hunk 6 above)
Comment (EpsilonPrime) [reply site/docs/index.md:27]: One of my pending PRs has a link to Substrait Fiddle which should cover that.
